ChangeLog

```diff
@@ -1,5 +1,18 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+   * Added config.h option MBEDTLS_TYPE_UDBL to allow configuring the
```

Comment:
`MBEDTLS_TYPE_UDBL` is currently not documented in `config.h`, and I'm not sure it belongs there because it's not a flag but a configurable value. 

Also, if we're introducing a configurable type `MBEDTLS_TYPE_UDBL`, we might consider having it overwrite our default choices, giving the user full flexibility even if there is compiler-support. 

_Concrete suggestion:_ Have two configuration options `MBEDTLS_NO_UDBL` and `MBEDTLS_FORCE_UDBL`. The meaning of `MBEDTLS_NO_UDBL` is that no double-width integer type should be used, regardless of the platform and compiler. The meaning of `MBEDTLS_FORCE_UDBL` is that some some user-specified type `MBEDTLS_TYPE_UDBL` for double-width integers has to be used, regardless of architecture and compiler. If neither `MBEDTLS_NO_UDBL` nor `MBEDTLS_FORCE_UDBL` are set (default), study compiler macros to detect the presence of double-width types. Regardless of the configuration, we set `MBEDTLS_HAVE_UDBL` to indicate whether some choice of double-width integers has been made.